extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -92,26 +109,28 @@ export default class JupyterServerInstallation {
 	private installPythonPackage(backgroundOperation: azdata.BackgroundOperation): Promise<void> {
 		let bundleVersion = constants.pythonBundleVersion;
 		let pythonVersion = constants.pythonVersion;
-		let packageName = 'python-#pythonversion-#platform-#bundleversion.#extension';
 		let platformId = utils.getOSPlatformId();
-
-		packageName = packageName.replace('#platform', platformId)
-			.replace('#pythonversion', pythonVersion)
-			.replace('#bundleversion', bundleVersion)
-			.replace('#extension', process.platform === constants.winPlatform ? 'zip' : 'tar.gz');
-
-		let pythonDownloadUrl: string = undefined;
-		switch (utils.getOSPlatform()) {
-			case utils.Platform.Windows:
-				pythonDownloadUrl = 'https://go.microsoft.com/fwlink/?linkid=2074021';
-				break;
-			case utils.Platform.Mac:
-				pythonDownloadUrl = 'https://go.microsoft.com/fwlink/?linkid=2065976';
-				break;
-			default:
-				// Default to linux
-				pythonDownloadUrl = 'https://go.microsoft.com/fwlink/?linkid=2065975';
-				break;
+		let packageName: string;
+		let pythonDownloadUrl: string;
+		if (this._usingExistingPython) {
+			packageName = `python-${pythonVersion}-${bundleVersion}-offlinePackages.zip`;
+			pythonDownloadUrl = 'https://go.microsoft.com/fwlink/?linkid=2086702';
```

Comment:
Minor: I'm wondering if all link IDs should go in a constants file? I can imagine we might need to update some day (e.g. breaking change time, where older versions shouldn't use newer links).

Reply:
Fixed.